src/providers/push/apns/apns.provider.ts

```diff
@@ -47,9 +47,13 @@ export class ApnsProvider implements Provider<any> {
     }
   }
   getMainNote(message: ApnsMessage) {
+    const seconds = 3600;
```

Comment:
```suggestion
    const expiresIn = 3600; // seconds
```
Just so that it's more clean on what those seconds are for